packages/backend-test-utils/src/next/wiring/TestBackend.ts

```diff
@@ -189,7 +189,11 @@ export async function startTestBackend<
 
   const backend = createSpecializedBackend({
     ...otherOptions,
-    services: [...factories, rootHttpRouterFactory, discoveryFactory],
+    defaultServiceFactories: [
+      ...factories,
```

Comment:
I think we'll want to drop factories all together here as an option, and install services using the `features` option instead, with `backend.add(...)`.

Reply:
What I think we do here is to just remove the `services` option